app/views/users/index.html.erb

```diff
@@ -1 +1,12 @@
-<h4>Here is a list of all users.</h4>
\ No newline at end of file
+<div class = "container">
+<% @user.each do |p| %>
+<div class = "user-row">
+    <div class = "photo">
+        <%= image_tag p.photo %>
+    </div>
+    <div class = "intro-details">
+       <p class = "user-name"> <%= p.name  %> </p>
+       <p class = "post-counter">Number of posts:  <%= p.posts_counter %> </p>
+    </div>    
+</div> 
```

Comment:
- [ ] Kindly take note that this code is inside a loop (@user.each do |p|). In Ruby, this loop should have an end statement. Let's fix it to avoid the syntax error message.